### PR TITLE
Fixing gpgkey and host_collection tests according to 6.2

### PR DESCRIPTION
**Pull request by oshtaier**
We have changes to alerts and now we have two types of them for gpgkeys
```
nosetests tests/foreman/ui/test_gpgkey.py -m content
..ESS......
======================================================================
ERROR: Hosts can install packages using gpg key associated with
----------------------------------------------------------------------
Ran 11 tests in 1395.997s

FAILED (SKIP=2, errors=1)
```
One failure related to another issue

### CODE REVIEW

File: robottelo/ui/locators.py [hunk 1]
```diff
@@ -1844,6 +1844,12 @@ def __iter__(self):
         ("//input[@placeholder='Filter' and contains(@ng-model, 'Search']")),
     "gpgkey.product_repo": (
         By.XPATH, "//td/a[contains(@href, 'repositories')]"),
+    "gpgkey.success_upload_alert": (
+        By.XPATH,
+        "//div[@ng-hide]//div[contains(@class, 'alert-success')]"),
+    "gpgkey.error_upload_alert": (
```
Comment (abalakh) [inline robottelo/ui/locators.py]: Xpath is pretty common here. Are you sure it's gpgkey-specific only? I wonder maybe the same alerts were added somewhere else so it's better to move them to `common_locators`?

**Comment by abalakh** [comment]
ACK pending question

**Comment by oshtaier** [comment]
@abalakh Yeap - that type of alerts is pretty rare, but you are right, they are not specific to gpgkey only. Gonna work on that

**Comment by oshtaier** [comment]
```
nosetests tests/foreman/ui/test_hostcollection.py
E.S...E.........
----------------------------------------------------------------------
Ran 16 tests in 2319.713s

FAILED (SKIP=1, errors=2)
```

**Comment by oshtaier** [comment]
Fixed `Hostcollection` entity in one scope

**Comment by oshtaier** [comment]
ready for review

**Comment by abalakh** [comment]
@oshtaier please update your commit message then.
ACK

**Comment by oshtaier** [comment]
Done

**Comment by elyezer** [comment]
ACK

**Pull request merged**